positron/src/positron/fastapi_template/app.py.tmpl

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+import asyncio
+import socket
+from threading import Thread
+
+import toga
+from asgiref.compatibility import guarantee_single_callable
+
+from .server import app as fastapi_app
+
+
+class SimpleReceive:
+    def __init__(self):
+        self._has_sent = False
+
+    async def __call__(self):
+        if not self._has_sent:
+            self._has_sent = True
+            return {{"type": "http.request", "body": b"", "more_body": False}}
+
+
+class {{{{ cookiecutter.class_name }}}}(toga.App):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    async def handle_client(self, reader, writer):
```

Comment:
It feels to me like this should be a standalone method (or class with a `send` member method), rather than a member of the App class.